crates/next-api/src/app.rs

```diff
@@ -1476,8 +1475,8 @@ impl AppEndpoint {
                         ..Default::default()
                     };
                     let edge_function_definition = EdgeFunctionDefinition {
-                        files: file_paths_from_root,
-                        wasm: wasm_paths_to_bindings(wasm_paths_from_root),
+                        files: file_paths_from_root.into_iter().collect(),
+                        wasm: wasm_paths_to_bindings(wasm_paths_from_root.into_iter().collect()),
```

Comment:
Should we sort these to make this more stable/deterministic?

Reply:
They are currently approximately in graph order, so they might indeed change. At least currently these lists changing more often isn't really a problem though.